src/traveler_dashboard_helper.js

```diff
@@ -17,12 +20,104 @@ function renderTripsHeader(title) {
   document.getElementsByTagName("BODY")[0].appendChild(upcomingTripsSection);
 }
 
+function renderAgentWelcome() {
+  document.getElementById("login-form").style.display = "none";
+  document.getElementById("footer").style.display = "none";
+  var welcomeHeader = document.createElement("H1");
+  var headerText = document.createTextNode("Welcome, AGENT");
+  welcomeHeader.appendChild(headerText);
+  document.body.appendChild(welcomeHeader);
+}
+
+function getPendingTrips(trips) {
+  let i;
+  let pendingTripsArr = [];
+  for (i = 0; i < trips.length; i++) {
+    if (trips[i].status === 'pending') {
+      pendingTripsArr.push(trips[i])
+    }
+  }
+  let pendingTrips = pendingTripsArr;
+  return pendingTrips;
+}
+
+function getLodgingCostsForAgent(destinations, trips) {
+  let approvedTripsArr = [];
+  let approvedTripsFlightPlusLodging = [];
+  let i;
+  for (i = 0; i < trips.length; i++) {
+    if (trips[i].status === "approved") {
+      approvedTripsArr.push(trips[i])
+    }
+  }
+
+  let j;
+  for (i = 0; i < approvedTripsArr.length; i++) {
+    for (j = 0; j < destinations.length; j++) {
```

Comment:
Is this possible with using i?  or does it overwrite every time?

I think if you can't get this to work in the classic way (using `i`), that you should refactor the inner loop to a for each so it is more readable 

at the very least change `j` to `destinationIndex` or something

Reply:
that way its not just an arbitrary letter.  `i` is conventionally used to signify `index`

Reply:
I was having trouble using 'i' again since it was throwing errors once it eventually became higher than the length of the destinations array..that makes a lot of sense to name it something relevant...gonna do that thanks.